src/services/spmNotification/index.ts

```diff
@@ -50,6 +69,7 @@ export const sendNotification = async (
       });
     } else {
       /* Send SPM notification to recipient (Only if enabled) */
+      const testData: TestData[] = getTestDataForNofityPdt(parsedFhirBundle);
```

Comment:
`TestData` type will be used for SPM notification?

Reply:
yes.. the old one api-notify api is still use this type in sns-notify package. if the cert include SER type, it will send old spm inbox notification.

Reply:
Oh yes... this one right?

https://github.com/Notarise-gov-sg/sns-notify-recipients/blob/dbd85fb9ef1139aac0ac60a7d14b8038bbc3e6ae/src/types.ts#L1-L7

Reply:
yes. but i'm already removed `TestData` type in this repo. just use sns package type.